src/component/handlers/edit/editOnSelect.js

```diff
@@ -20,10 +20,8 @@ var getDraftEditorSelection = require('getDraftEditorSelection');
 import type DraftEditor from 'DraftEditor.react';
 
 function editOnSelect(editor: DraftEditor): void {
-  if (editor._blockSelectEvents) {
-    return;
-  }
-  if (editor._latestEditorState !== editor.props.editorState) {
+  if (editor._blockSelectEvents ||
```

Comment:
Checking these two conditions is insufficient; I observe this event handler called when pressing two different keys in quick succession which is why I added an extra check in #919.